lib/applications.js

```diff
@@ -60,6 +65,19 @@ class Application {
     const appInterface = this.api[interfaceName];
     return appInterface && Object.keys(appInterface);
   }
+
+  // Handle incoming event
+  //   connection - JSTP connection that received event
+  //   interfaceName - name of the interface
+  //   eventName - name of the event
+  //   args - event arguments
+  //
+  handleEvent(connection, interfaceName, eventName, args) {
+    if (this.eventHandlers) {
+      const handler = this.eventHandlers[interfaceName][eventName];
```

Comment:
If there is no object available under `interfaceName` key, this line will throw, I think.